static/javascript/launch.js

```diff
@@ -400,14 +400,12 @@ function updateSDSDropdown() {
   const surveyId = schemaSurveyId;
   const periodId = document.getElementById("period_id")?.value;
 
-  const supplementaryDataSection = document.querySelector(
-    "#supplementary_data",
-  );
+  const supplementaryDataSection = document.querySelector("#sds_id");
```

Comment:
I think the id was more descriptive as it was. I dont think "sds_id" is a good id because it sort of describes what it is rather than setting it to an actual id that makes sense if Ive explained that right